src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.spring.controller;
+
+import javax.validation.Valid;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService service;
+
+    public AuthenticationController(AuthenticationService service) {
```

Comment:
```suggestion

    @Autowired
    public AuthenticationController(AuthenticationService service) {
```

Lets make our code more consistent, lets use annotation @Autowired over all constructors(or nowhere)